src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -136,6 +137,8 @@ module CNVegCarbonFluxType
      real(r8), pointer :: frootc_to_litter_patch                    (:)     ! fine root C litterfall (gC/m2/s)
      real(r8), pointer :: livestemc_to_litter_patch                 (:)     ! live stem C litterfall (gC/m2/s)
      real(r8), pointer :: repr_grainc_to_food_patch               (:,:)     ! grain C to food for prognostic crop(gC/m2/s) [patch, repr_grain_min:repr_grain_max]
+     real(r8), pointer :: repr_grainc_to_food_perharv             (:,:,:)   ! grain C to food for prognostic crop accumulated by harvest (gC/m2) [patch, harvest, repr_grain_min:repr_grain_max]
+     real(r8), pointer :: repr_grainc_to_food_thisyr              (:,:)     ! grain C to food for prognostic crop accumulated this calendar year (gC/m2) [patch, repr_grain_min:repr_grain_max]
```

Comment:
Resolved second issue with fe257d4. They're not per-second because these variables represent accumulations over each growing season (`_perharv`) or calendar year (`_thisyr`), to be saved instantaneously at the end of each calendar year. The idea is to provide output that's easier to work with than the per-second flux values that must be summed up over each growing season from daily output files.

Reply:
Resolved first issue with 6bf99fa75.